mhs/inbound/inbound/request/tests/test_handler.py

```diff
@@ -106,6 +107,32 @@ def test_correct_workflow(self):
 
         self.assertEqual(ack_response.code, 200)
 
+    @unittest.mock.patch.object(message_utilities.MessageUtilities, "get_timestamp")
+    @unittest.mock.patch.object(message_utilities.MessageUtilities, "get_uuid")
+    def test_workflow_throws_exception(self, mock_get_uuid, mock_get_timestamp):
+        self.mock_workflow.handle_inbound_message.side_effect = Exception("what a failure")
+        mock_get_uuid.return_value = "5BB171D4-53B2-4986-90CF-428BE6D157F5"
+        mock_get_timestamp.return_value = "2012-03-15T06:51:08Z"
+        request_body = file_utilities.FileUtilities.get_file_string(str(self.message_dir / REQUEST_FILE))
+
+        response = self.fetch("/", method="POST", body=request_body, headers=CONTENT_TYPE_HEADERS)
+
+        self.assertEqual(response.code, 500)
+        expected = b'<html><title>500: Exception in workflow</title><body>500: ' \
+                   b'Exception in workflow</body></html>'.decode('utf-8')
```

Comment:
There's no point hardcoding a bytestring that you then decode when you can just do a normal string in the first place.